### PR TITLE
fix: delete sim_main in `__init__` all

### PR DESCRIPTION
**Pull request by yihong0618**
caused it never use since sim_main had been deteled

### CODE REVIEW

**Comment by gaogaotiantian** [comment]
Thanks! Do you mind add a regression test in `test_cmdline.py` to test `from viztracer import *`?

**Comment by yihong0618** [comment]
> Thanks! Do you mind add a regression test in `test_cmdline.py` to test `from viztracer import *`?

will do that today

**Comment by yihong0618** [comment]
> Thanks! Do you mind add a regression test in `test_cmdline.py` to test `from viztracer import *`?

Done

File: tests/test_cmdline.py [hunk 1]
```diff
@@ -379,6 +379,9 @@ def test_pid_suffix_and_output(self):
     def test_module(self):
         self.template(["viztracer", "-m", "numbers"])
 
+    def test_import_star(self):
+        self.template(["python", "-c", "from viztracer import *"], expected_output_file=None)
```
Comment (gaogaotiantian) [inline tests/test_cmdline.py:383]:
```suggestion
        self.template(["python", "-c", "from viztracer import *"], script=None,  expected_output_file=None)
```

File: tests/test_cmdline.py [hunk 2]
```diff
@@ -379,6 +379,9 @@ def test_pid_suffix_and_output(self):
     def test_module(self):
         self.template(["viztracer", "-m", "numbers"])
 
+    def test_import_star(self):
+        self.template(["python", "-c", "from viztracer import *"], script=None, expected_output_file=None)
```
Comment (gaogaotiantian) [inline tests/test_cmdline.py:383]:
```suggestion
        script = "from viztracer import *"
        self.template(["python", "cmdline_test.py"], script=script, expected_output_file=None)
```

**Comment by gaogaotiantian** [comment]
Thank you for noticing this!

**Pull request merged**